hypothesis-python/src/hypothesis/internal/entropy.py

```diff
@@ -64,7 +65,7 @@ def get_seeder_and_restorer(seed=0):
     using the global random state.  See e.g. #1709.
     """
     assert isinstance(seed, int) and 0 <= seed < 2 ** 32
-    states = []  # type: list
+    states: List[Tuple[int, ...]] = []
```

Comment:
Not true in general; the state for e.g. Numpy PRNG is quite different.  `List[Any]`, or just `list`, is more accurate here.